tests/ProjectJoinRequestsTest.php

```diff
@@ -0,0 +1,609 @@
+<?php
+namespace Keboola\ManageApiTest;
+
+use Keboola\ManageApi\ClientException;
+
+class ProjectJoinRequestsTest extends ClientTestCase
+{
+    private $organization;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
+        ]);
+
+        $this->client->addUserToOrganization($organization['id'], ['email' => $this->normalUser['email']]);
```

Comment:
Myslím si že by bylo lepší v tom setupu udělat jenom čistku. Tzn. všechny odebrat a nikoho nepřidávat.
A  až pak v konkrétních testech to připravit jak je zrovna potřeba. Aby člověk při čtení testu pořád nemusel řešit jestli tenhle user teda je v té organizaci nebo ne apod. To že to v tech testech pak často bude duplicitně imho nevadí, důležitější je ať je na první pohled vidět jak je to připravené.